eng/common/matrix/Create-SparseJobMatrix.ps1

```diff
@@ -0,0 +1,23 @@
+<#
```

Comment:
instead of having 2 entry point scripts should we just make it a parameter?

Reply:
I'll do that. This whole entry point is going to go away as well once I add support (which all/sparse will be implemented through).